src/Storage/Repository/PgInstanceRepository.cs

```diff
@@ -20,11 +20,12 @@ namespace Altinn.Platform.Storage.Repository
     /// </summary>
     public class PgInstanceRepository: IInstanceRepository
     {
+        private const string _readSqlFilteredInitial = "select * from storage.readinstancefromquery_v2 (";
         private readonly string _deleteSql = "select * from storage.deleteinstance ($1)";
         private readonly string _insertSql = "call storage.insertinstance ($1, $2, $3, $4, $5, $6, $7, $8)";
         private readonly string _upsertSql = "call storage.upsertinstance ($1, $2, $3, $4, $5, $6, $7, $8)";
         private readonly string _readSql = "select * from storage.readinstance ($1)";
-        private readonly string _readSqlFiltered = "select * from storage.readinstancefromquery (";
+        private readonly string _readSqlFiltered = _readSqlFilteredInitial;
```

Comment:
Could the sql be defined directly on this lin instead?

Reply:
This is to avoid having and maintaining a very long string literal in the code.